indico/queries/datasets.py

```diff
@@ -215,6 +215,7 @@ def __init__(
         name: str,
         files: List[str],
         wait: bool = True,
+        max_wait_time: Tuple[int, float] = 5,
```

Comment:
Is it okay to add here or should this be at bottom of kwarg list

Reply:
good call - bottom of list - just in case we got some folks who blindly pass params

Reply:
and same Union, not Tuple comment